ModelBundle/DataFixtures/MongoDB/DemoContent/HomeRootDataGenerator.php

```diff
@@ -0,0 +1,176 @@
+</div>
+EOF;
+        $routePattern = "en";
+        $language = "en";
+
+        return $this->generateNodeGlobal($htmlContent, $language, $routePattern);
+    }
+
+    /**
+     * @return Node
+     */
+    protected function generateNodeDe()
+    {
+        $htmlContent = <<<EOF
+<div class='content2'>
+    <h1>Open-Orchestra</h1>
+    <p>Offene Orchestra ist ein leistungsfähiges Web-Integrationsplattform zur Beschleunigung der Bau von mehrjährige digitale Ökosysteme.
+     Diese Lösung von Interakting Erfahrung in der Entwicklung der internationalen Plattformen ist verfügbar unter der Open Source Lizenz.</p>
+    <p>In strikter Übereinstimmung mit den Standards und Best Practices Framework entwickelt auf Symfony2 und MongoDB.
+    Offene Orchestra ist schnell, sehr anpassungsfähig und erweiterbar, Multi-Site und Multi-Geräte.</p>
+    <p>Offene Orchestra CMS bietet erweiterte Funktionen und SI internen und externen Integrationskomponenten
+    in hohem Maße konfigurierbar, modular, hohen Belastungen und Sicherheit zugeschnitten.</p>
+    <p>Eine gezielte Lösung :
+    <ul>
+        <li>Projekt, wo «Erfahrung», ob Kunde, Mitarbeiter, Partner oder Händler ist der Kern des Problems.</li>
+        <li>Internationale Projekt Skaleneffekte erfordern.</li>
+        <li>Komplexe Projekte, bei denen interne Informationssysteme und Partner sind hoch beanspruchte.</li>
+        <li>Projekt, dessen Ziel ist es, digitale Ökosysteme (E-Commerce, Kommunikation, Referenz, Selbst-Pflege, Mobilität, Vertrieb, ...),
+        die mit den funktionalen und technologischen Synergien aufzubauen.</li>
+    </ul></p>
+    <p>Unser versprechen : « Skaleneffekte und die Bündelung von Investitionen für eine konsistente Web-Erfahrung auf allen Kanälen Fixed Mobile,
+    Tablet, TV, Poller ... »</p>
+</div>
+EOF;
+        $routePattern = "de";
+        $language = "de";
+
+        return $this->generateNodeGlobal($htmlContent, $language, $routePattern);
+    }
+
+    /**
+     * @param string $htmlContent
+     * @param string $language
+     * @param string $routePattern
+     *
+     * @return Node
+     */
+    protected function generateNodeGlobal($htmlContent, $language, $routePattern)
+    {
+        $siteHomeBlock0 = new Block();
+        $siteHomeBlock0->setLabel('Wysiwyg');
+        $siteHomeBlock0->setComponent(TinyMCEWysiwygStrategy::NAME);
+        $siteHomeBlock0->setAttributes(array(
+            "htmlContent" => $htmlContent
+        ));
+        $siteHomeBlock0->addArea(array('nodeId' => 0, 'areaId' => 'mainContentArea1'));
+
+        $siteHomeArea4 = $this->createArea('Main content area 1', 'mainContentArea1', 'main-content-area1');
+        $siteHomeArea4->addBlock(array('nodeId' => 0, 'blockId' => 1));
+        $siteHomeArea5 = $this->createModuleArea();
+        $siteHomeArea3 = $this->createMain(array($siteHomeArea4, $siteHomeArea5));
+        $siteHomeArea6 = $this->createFooter();
+
+        if($this->version > 1) {
```

Comment:
because there is a reference anyway (the root node). to test on reference i will have to make the difference between the name of a normal node or  a version one and to do that i will have to duplicate the function addNode in LoadNodeGlobalDemoData.php in which i set the reference. so to avoid duplicate this function i test on version.